edgetest/core.py

```diff
@@ -105,31 +129,85 @@ def setup(
                 pkg += f"[{', '.join(extras)}]"
             if deps:
                 LOG.info(
-                    f"Installing specified additional dependencies into {self.envname}..."
+                    "Installing specified additional dependencies into %s: %s",
+                    self.envname,
+                    ", ".join(deps),
```

Comment:
Can we use f-strings for consistency? Same throughout.

```suggestion
                    f"Installing specified additional dependencies into {self.envname}: {', '.join(deps)}"
```

Reply:
Did this because of this [small reason](https://docs.astral.sh/ruff/rules/logging-f-string/). I know it's not consistent, but didn't want to change all logging statements. Probably doesn't matter too much either. Lmk!

Reply:
Oh I haven't seen that before! We should probably align to that model then.